README.md

```diff
@@ -76,3 +76,24 @@ Then configure the rules you want to use under the rules section.
 * [`typescript/prefer-namespace-keyword`](./docs/rules/prefer-namespace-keyword.md) — Require the use of the `namespace` keyword instead of the `module` keyword to declare custom TypeScript modules. (`no-internal-module` from TSLint)
 * [`typescript/type-annotation-spacing`](./docs/rules/type-annotation-spacing.md) — Require consistent spacing around type annotations
 <!-- end rule list -->
+
+## Contributing
+
+For a PR to be merged, all tests must pass, the must be no lint errors, and the code must be formatted.
+- `yarn test`
```

Comment:
Technically `yarn test` will run `yarn lint` for you as it has a `pretest` script.
But I'm ok leaving it as is... emphasizes the importance of running both :smile:...

Reply:
I just changed the config so that doesn't to make it a bit easier to run tests.
Gets a bit annoying when you're developing a rule to have to wait for the lint + docs check as well.

I've changed the both the precommit hook and travis so they explicitly run all of the checks.

(test run with the new config: https://travis-ci.org/bradzacher/eslint-plugin-typescript/jobs/460887254)